web/environments.php

```diff
@@ -6,10 +6,18 @@
         define('ENVIRONMENT', 'production');
         break;
```

Comment:
Thanks @danielogen , could we be having a default configuration for cases that we might not be able to capture here? Can those cases be handled from here or we have to explicitly look out for them? Thanks.

Reply:
Thank you @Baalmart , the default case is actually the `development` environment